### PR TITLE
Dev did method key

### PR DESCRIPTION
**Pull request by mattcollier**

### CODE REVIEW

**Comment by mattcollier** [comment]
Putting in PR for discussion purposes, needs some clean-up if we're keeping this.

**Comment by dlongley** [comment]
I'm going to look into this.

**Comment by dlongley** [comment]
@mattcollier -- Please take a look. If you approve, this can be merged.

File: lib/caches.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+// load config defaults
+require('./config');
+
+// module API
+const api = {};
+module.exports = api;
+
+bedrock.events.on('bedrock.init', async () => {
+  const cfg = bedrock.config.kms;
+  api.keyRecords = new LRU({max: cfg.keyRecordCache.maxSize});
```
Comment (mattcollier) [inline lib/caches.js:18]: why is `cfg.keyRecordCache.maxAge` not used here as well.  I see that maxAge is being specified on the cache `set` calls, which is valid, but why?

Comment (dlongley) [reply lib/caches.js:18]: It was an oversight that you could pass in a default maxAge. I'll update it.

Comment (dlongley) [reply lib/caches.js]: Updated.

**Comment by mattcollier** [comment]
Approved.

**Review by dlongley — APPROVED** [review]

**Pull request merged**